Runtimes/Supplemental/Distributed/CMakeLists.txt

```diff
@@ -57,47 +72,53 @@ option(${PROJECT_NAME}_ENABLE_LIBRARY_EVOLUTION "Generate ABI resilient runtime
 option(${PROJECT_NAME}_ENABLE_PRESPECIALIZATION "Enable generic metadata prespecialization"
   ${SwiftCore_ENABLE_PRESPECIALIZATION})
 
-add_library(swiftDistributed
-  DistributedActor.cpp
-  DistributedActor.swift
-  DistributedActorSystem.swift
-  DistributedAssertions.swift
-  DistributedDefaultExecutor.swift
-  DistributedMacros.swift
-  DistributedMetadata.swift
-  LocalTestingDistributedActorSystem.swift)
-
-target_compile_options(swiftDistributed PRIVATE
+add_compile_options(
```

Comment:
Why are we switching to `add_compile_options` here?

Reply:
To keep in sync with the other modules that we are building. It makes diffing the CMakeLists.txt easier.